line-bot-api-client/src/main/java/com/linecorp/bot/client/LineMessagingServiceBuilder.java

```diff
@@ -201,12 +202,13 @@ public LineMessagingService build() {
     }
 
     private static Retrofit.Builder createDefaultRetrofitBuilder() {
-        final ObjectMapper objectMapper = new ObjectMapper();
-        objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
-
-        // Register JSR-310(java.time.temporal.*) module and read number as millsec.
-        objectMapper.registerModule(new JavaTimeModule())
-                    .configure(DeserializationFeature.READ_DATE_TIMESTAMPS_AS_NANOSECONDS, false);
+        final ObjectMapper objectMapper = new ObjectMapper()
+                .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
+                // Register ParameterNamesModule to read parameter name from lombok generated constructor.
+                .registerModule(new ParameterNamesModule())
```

Comment:
I guess ParameterNamesModule is no longer required.

Reply:
ParameterNamesModule is used to read mapping names from constructor name.
(Suppressing `@JsonParameter` annotation is possible by this)

I think supressing writing fieldname twice is good.

Can I use ParameterNamesModule?